### ISSUE TITLE
Jamovi does not upgrade to new current version without --pre (1.1.8.0-current)

### ISSUE THREAD

**Issue by Gootjes** [OlafHaag/chocolatey-packages#17]
I am using `choco upgrade jamovi -y` to update Jamovi, but it failed.
I had to specify `--pre` to make it work.
Maybe this will happen only once because of the changes due to #16 .
But I am reporting it now anyway as a heads up for the next version update. Let's see if it happens again!

```
Chocolatey v0.10.15
Upgrading the following packages:
jamovi
By upgrading you accept licenses for the packages.

You have jamovi v1.1.7.000-current installed. Version 1.1.8.0-current is available based on your source(s).
jamovi not upgraded. An error occurred during installation:
 Unable to resolve dependency 'jamovi.install (= 1.1.8.0-current)'.
jamovi package files upgrade completed. Performing other installation steps.
The upgrade of jamovi was NOT successful.
jamovi not upgraded. An error occurred during installation:
 Unable to resolve dependency 'jamovi.install (= 1.1.8.0-current)'.

Chocolatey upgraded 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - jamovi (exited 1) - jamovi not upgraded. An error occurred during installation:
 Unable to resolve dependency 'jamovi.install (= 1.1.8.0-current)'.
```

**Comment by OlafHaag** [comment]
I believe that is supposed to happen. You have to specify `--pre` in order to install the pre-releases. 
But it's strange that it notifies you about a newer version. I would have presumed that choco would only look for stable releases if --pre is not specified.
Anyway, I don't think there's much I can do here.

This could also happen with bad timing. When there's a newer version available the meta package gets pushed first and queued to the verification process. After that the install and portable packages get pushed and queued. I always get this error from the verification, since the install packages is verified and listed only after the meta package. I then rerun the tests. It would be better if the install package would get pushed before the meta package.

**Comment by Gootjes** [comment]
It happened again, maybe I need to wait for a few days more?

It is still kind of odd, are you sure you are updating both streams `current` and `solid` by using the `PushAll = $true` option for `updateall`?

See https://github.com/majkinetor/au#streams


Nevermind, I do not understand, the api does show the version 1.2.0.0-current exists:
https://chocolatey.org/api/v2/FindPackagesById?id=%27jamovi.install%27&$skiptoken=%27jamovi.install%27,%270.9.1.1100%27

But I do get an error like this:
```console
C:\WINDOWS\system32>choco upgrade jamovi
Chocolatey v0.10.15
Upgrading the following packages:
jamovi
By upgrading you accept licenses for the packages.
You have jamovi v1.1.8.0-current installed. Version 1.2.0.0-current is available based on your source(s).
jamovi not upgraded. An error occurred during installation:
Unable to resolve dependency 'jamovi.install (= 1.2.0.0-current)'.
jamovi package files upgrade completed. Performing other installation steps.
The upgrade of jamovi was NOT successful.
jamovi not upgraded. An error occurred during installation:
Unable to resolve dependency 'jamovi.install (= 1.2.0.0-current)'.
Unsuccessful operation for jamovi.
Rollback to previous version (package files only)?([Y]es/[N]o):

```

I uninstalled, and installed it just now using --pre, and that works, but upgrading from a previous version does not seem to work.
To summarize:
Not working:
```console
choco upgrade all
choco upgrade jamovi
```
Working:
```console
choco upgrade jamovi --pre
choco uninstall jamovi; choco install jamovi --pre
```

**Comment by OlafHaag** [comment]
That's really odd. Both streams where updated at the same time. Since you were on _v1.1.8.0-current_ `cup jamovi` should have at least updated to _v1.1.9.0_ solid stream when `--pre` wasn't specified.
I can only imagine that the verification process of the v1.2.0.0-current package wasn't done yet, but then it shouldn't be listed as an available update.

Unfortunately I'm at a loss here and would need help from the choco devs to clear this up.

**Comment by Gootjes** [comment]
I reported it [here ](https://github.com/chocolatey/choco/issues/1957)as the same issue has been reported with another package, let's hope there is an easy fix!

**Comment by OlafHaag** [comment]
Thanks a lot for taking the time to track down the issue and reporting it. I appreciate it!
